### PR TITLE
Add edac-utils package

### PR DESCRIPTION
**Pull request by L38s**
# Purpose of pull request

edac_utils

This pr adds edac_utils package based on meta-openembedded's /meta-networking/recipes-connectivity/firewalld/firewalld_0.6.3.bb

# Test
## How to test

Check the existence of the component.
```
root@qemuarm64:~# find / -name "libedac*" -type f
root@qemuarm64:~# which edac-util
root@qemuarm64:~# which edac-ctl
```
Execute the edac-util and edac_core commands.

## Test result

Find Components
```
root@qemuarm64:~# find / -name "libedac*" -type f
/usr/lib/libedac.so.1.1.1
root@qemuarm64:~# which edac-util
/usr/bin/edac-util
root@qemuarm64:~# which edac-ctl
/usr/sbin/edac-ctl
```
### edac-util
```
root@qemuarm64:~# edac-util
edac-util: Error: No memory controller data found.
root@qemuarm64:~# edac-util -v
edac-util: Error: No memory controller data found.
root@qemuarm64:~# edac-util -h
Usage: edac-util [OPTIONS]
  -h, --help           Display this help
  -q, --quiet          Display only non-zero error counts and fatal errors
  -v, --verbose        Increase verbosity. Multiple -v's may be used
  -s, --status         Display EDAC status
  -r, --report=REPORT  Display EDAC error report REPORT

Valid REPORT types are default, simple, full, ue, ce

root@qemuarm64:~# edac-ctl --status
[  204.898364] random: edac-ctl: uninitialized urandom read (4 bytes read)
edac-ctl: drivers not loaded.
```
### edac-ctl
```
root@qemuarm64:~# edac-ctl
[   46.142061] random: edac-ctl: uninitialized urandom read (4 bytes read)
Usage: edac-ctl [OPTIONS...]
 --quiet            Quiet operation.
 --mainboard        Print mainboard vendor and model for this hardware.
 --status           Print status of EDAC drivers.
 --print-labels     Print Motherboard DIMM labels to stdout.
 --register-labels  Load Motherboard DIMM labels into EDAC driver.
 --delay=N          Delay N seconds before writing DIMM labels.
 --labeldb=DB       Load label database from file DB.
 --help             This help message.
root@qemuarm64:~# edac-ctl --status
[  260.924662] random: edac-ctl: uninitialized urandom read (4 bytes read)
edac-ctl: drivers not loaded.
```

### CODE REVIEW

File: recipes-debian/edac-utils/edac-utils_debian.bb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# base recipe: meta-openembedded/meta-networking/recipes-connectivity/firewalld/firewalld_0.6.3.bb
```
Comment (masami256) [inline recipes-debian/edac-utils/edac-utils_debian.bb]: It shouldn't based on firewalld, right?

**Comment by masami256** [comment]
> This pr adds edac_utils package based on meta-openembedded's /meta-networking/recipes-connectivity/firewalld/firewalld_0.6.3.bb

It shouldn't based on firewalld, right?

File: recipes-debian/edac-utils/edac-utils_debian.bb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+"
+
+inherit autotools-brokensep
+
+do_configure_prepend () {
+    touch ${S}/ChangeLog
+    ${S}/bootstrap
+}
+
+RDEPENDS_${PN}_x86 = "dmidecode"
```
Comment (masami256) [inline recipes-debian/edac-utils/edac-utils_debian.bb:36]: aarch64 and other architectures depend on dmidecode package, do we need dmidecode package too?

Comment (L38s) [reply recipes-debian/edac-utils/edac-utils_debian.bb:36]: We will use dmidecoded in poky.
This time, the decision is that it will not be used at runtime.

Comment (masami256) [reply recipes-debian/edac-utils/edac-utils_debian.bb:36]: Okay. I got it.

File: recipes-debian/edac-utils/edac-utils_debian.bb (hunk 1 above)
Comment (L38s) [reply recipes-debian/edac-utils/edac-utils_debian.bb]: I made a mistake. I'll fix it.

**Comment by masami256** [comment]
Thanks !

**Comment by masami256** [comment]
Tested on LS1046A-RDB. Set ``` CONFIG_EDAC_LAYERSCAPE=m ``` and build image.
When layerscape_edac_mod is loaded, you will see error report by edac-util command.

```
root@ls1046ardb:~# lsmod
    Not tainted
ipv6 430080 18 [permanent], Live 0xffff000000bcb000
crc32_ce 16384 0 - Live 0xffff000000bab000
crct10dif_ce 16384 0 - Live 0xffff000000b8b000
lm90 28672 0 - Live 0xffff000000b7f000
pci_endpoint_test 20480 0 - Live 0xffff000000b6d000
ina2xx 16384 0 - Live 0xffff000000b59000
layerscape_edac_mod 20480 0 - Live 0xffff000000b50000
root@ls1046ardb:~# edac-util -sv
edac-util: EDAC drivers are loaded. 1 MC detected:
  mc0:fsl_mc_err

root@ls1046ardb:~# edac-util -r
edac-util: No errors to report.
root@ls1046ardb:~# 
```

**Comment by hiraku-wfs** [comment]
@masami256 Thank you for the testing!

**Pull request merged**